portfolio/src/main/webapp/comments.html

```diff
@@ -76,6 +76,17 @@ <h1>Comments</h1>
             <option value="dsc" selected>Date: new to old</option>
             <option value="asc">Date: old to new</option>
           </select>
+          <label for="lang">Language:</label>
```

Comment:
As I understand it, the "for" attribute is supposed to match up with the ID of an input. But that doesn't seem to be the case here. Why do you have 'for="lang"'?